syntax = "proto3";
package ni.measurementlink.measurement.tests;

message NIDCPowerConfigurations {
    repeated string pin_names = 1;
    bool multi_session = 2;
}

message NIDCPowerOutputs {
    repeated string session_names = 1;
    repeated string resource_names = 2;
    repeated string channel_lists = 3;
    repeated string channels_connected = 4;
}
